### ISSUE TITLE
Admin Import Table Lacks Accessible Design, Especially for Checkboxes

### ISSUE THREAD

**Issue by Santoz2** [pressbooks/pressbooks#1791]
### Description

While there is an existing issue for the Admin Organize tables (see https://github.com/pressbooks/pressbooks/issues/1780), the implementation of this Import table is a bit different. In particular, the behaviors of the checkboxes in this table are peculiar.

### Steps to Reproduce

1. Navigate to the Tools/Import tab of the Admin interface Main Menu
2. Using Microsoft Word, create a document with at least two headings
3. Import your newly made Word document by selecting Microsoft Word for the Import Type, fining your file with the "Browse..." button, and pressing Begin Import
4. Navigate to the Table on the Import page. Note how the first item is an unchecked Checkbox with a hidden "Import" label. Note how this label gives no indication that the checkbox selects all of the rows below in the table.
5. Select the checkbox, note how there is no indication that the checkbox and the checkboxes below it have been checked.
6. Navigate to the next row (the first non-header row). Note how the label for this checkbox is the content of the second cell, the Title of a section. Note how there is no row header for this row.
7. Navigate to the second column in the row, the Title column. Note how this cell is clickable, and activate it. Note how the checkbox in the first column is checked, and focus is brought to that cell.

**Expected behavior:** The rows of the Import table should have row headers. The cells of the Title column may work well for this, but they are currently being used as the labels for the checkboxes. It should be clear that the checkbox in the top left corner selects all Titles for import, which may be achieved by changing the label of that checkbox to "Import all" (which is coincidentally the id for that checkbox). 

**Actual behavior:** Placing the label of a checkbox in a different row than the actual checkbox produces some odd behaviors. Activating the checkbox does not announce that it has been checked or not checked. Activating the label for the checkbox moves focus to the column containing the checkbox, which can be very confusing. It is not clear that the top left checkbox imports all items, as its label only says "Import."

**Comment by SteelWagstaff** [comment]
This table https://github.com/pressbooks/pressbooks/blob/d9e7efc1828bbd13adaaa8972c3fcac09a9706c3/templates/admin/import.php#L79-L126 could use similar a11y fixes to those applied to resolve https://github.com/pressbooks/pressbooks/issues/1633

**Comment by avaaec** [comment]
This is a UW ATS priority 2 issue
